### ISSUE TITLE
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult'

### ISSUE THREAD

**Issue by sravimohan** [PowerShell/PSReadLine#3024]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19042
BufferWidth: 120
BufferHeight: 29

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![image](https://user-images.githubusercontent.com/253584/141582172-9b9afcff-6680-4bb3-b88c-b5288dd3e7a5.png)


### Environment data

```console
PS Version: 5.1.19041.1320
PS HostName: ConsoleHost (Windows Terminal)
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 201
BufferHeight: 45
```

### Steps to reproduce

update to v7.2.0 and launch

### Expected behavior

able to run Powershell 7

### Actual behavior

error on launching powershell

**Comment by andyleejordan** [comment]
Hi, I see you're using an old version of PSReadLine. Please upgrade to use the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4) and let us know if this continues to reproduce. Instructions for doing so:

1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:

1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.

Thanks!

**Comment by sravimohan** [comment]
updating PSReadLine fixed the issue
